ambry-rest/src/test/java/com.github.ambry.rest/NettyResponseChannelTest.java

```diff
@@ -58,6 +62,25 @@
  * {@link MockNettyMessageProcessor#handleContent(HttpContent)}
  */
 public class NettyResponseChannelTest {
+  private static final Map<RestServiceErrorCode, HttpResponseStatus>
+      REST_SERVICE_ERROR_CODE_TO_HTTP_RESPONSE_STATUS_MAP = new HashMap<>();
```

Comment:
Can we name this as "REST_ERROR_CODE_TO_HTTP_STATUS". name seems to be too long.